hangman.py

```diff
@@ -0,0 +1,129 @@
+import random
+from words import words
+
+def hangman_pick_word():
```

Comment:
The function returns the word and theme so the function name should be updated accordingly.